app/actions/remote/groups.ts

```diff
@@ -57,12 +58,19 @@ export const fetchGroupsForChannel = async (serverUrl: string, channelId: string
 export const fetchGroupsForTeam = async (serverUrl: string, teamId: string, fetchOnly = false) => {
     try {
         const {operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+
         const client: Client = NetworkManager.getClient(serverUrl);
         const response = await client.getAllGroupsAssociatedToTeam(teamId);
 
-        return operator.handleGroups({groups: response.groups, prepareRecordsOnly: fetchOnly});
+        const groups = await operator.handleGroups({groups: response.groups, prepareRecordsOnly: true});
+        const groupTeams = await operator.handleGroupTeamsForTeam({groups: response.groups, teamId, prepareRecordsOnly: true});
```

Comment:
could these two be called in parallel ?

Reply:
Yup - changed to `Promise.all()`, and in the other methods too 👍